examples/json-record-map-conversion/json_record_map_conversion.bal

```diff
@@ -14,6 +14,10 @@ type Movie record {
     Person writer;
 };
 
+type j1 json;
+
+type map1 map<anydata>;
```

Comment:
Better if we do not name types like this. Usually, a type name starts with a capital letter and don't use numbers. We use names that convey what that type is about. How about something like `MapAnydata`?